src/assets/scss/_breakpoints.scss

```diff
@@ -0,0 +1,55 @@
+// Media Queries ///////////////
+
+@mixin breakpoint($class) {
+  @if $class==xs {
+    @media (max-width: 767px) {
+      @content;
+    }
+  }
+
+  @if $class==xsm {
+    @media (min-width: 600px) {
+      @content;
+    }
+  }
+
+  @else if $class==sm {
+    @media (min-width: 768px) {
+      @content;
+    }
+  }
+
+  @else if $class==menu {
+    @media (min-width: 855px) {
+      @content;
+    }
+  }
+
+  @else if $class==md {
+    @media (min-width: 992px) {
+      @content;
+    }
+  }
+
+  @else if $class==lg {
+    @media (min-width: 1200px) {
+      @content;
+    }
+  }
+
+  @else if $class==container {
+    @media (min-width: 84.375em) {
+      @content;
+    }
+  }
+
+  @else if $class==xlg {
+    @media (min-width: 106em) {
+      @content;
+    }
+  }
+
+  @else {
+    @warn "Breakpoint mixin supports: xs, sm, md, lg";
```

Comment:
Should this include the other options that are supported?

Reply:
Good point!